driver/sdk.md

```diff
@@ -1,4 +1,97 @@
 
 # Babelfish SDK
 
-* **TODO**
\ No newline at end of file
+The [Babelfish SDK](https://github.com/bblfsh/sdk/) contains tools and libraries
+required to create a Babelfish driver.
+
+## Prerequisites
+
+Babelfish has the following prerequisites:
+
+* [Docker](https://www.docker.com/get-docker)
+* [Go](https://golang.org/dl/)
+
+## Installing
+
+Install the Babelfish SDK:
+
+```bash
+$ go get github.com/bblfsh/sdk/...
```

Comment:
Well, that's a good point. We should discuss if we want to version the SDK. Anyway, this documentation reflects the current state of the project.